### ISSUE TITLE
Basic search in top bar

### ISSUE THREAD

**Issue by patzick** [DivanteLtd/shopware-pwa#466]
## Blocked by
#465

## Acceptance criteria
- [x] create composable for searching using API client (Separate PR)
- [x] add search inputs from SFUI to topbar for desktop
- [x] implement search input on mobile view
- [x] invoke search in composable

In scope of this PR there is no visualization of search results, they can be displayed only in console for example.
